src/mapbox.android.ts

```diff
@@ -2627,7 +2627,7 @@ export class Mapbox extends MapboxCommon implements MapboxApi {
     addSource(id: string, options: AddSourceOptions, nativeMap?): Promise<void> {
         return new Promise((resolve, reject) => {
             try {
-                const { url, type } = options;
+                const { type } = options;
```

Comment:
i would suggest that if you are on your way to clean up unused variables, get rid of this one too (only gets used in a console.log, which can use options.type) :-)